pkg/image/common/validator.go

```diff
@@ -186,6 +203,21 @@ func (v *vmiValidator) CheckSecurityParameters(vmi *v1beta1.VirtualMachineImage)
 	return nil
 }
 
+// CheckPVCInUse checks if the PVC is in use by any pods, this is only used to CDI backend
+func (v *vmiValidator) CheckPVCInUse(vmi *v1beta1.VirtualMachineImage) error {
+	index := fmt.Sprintf("%s-%s", vmi.Spec.PVCNamespace, vmi.Spec.PVCName)
+	if pods, err := v.podCache.GetByIndex(util.IndexPodByPVC, index); err == nil && len(pods) > 0 {
+		podList := []string{}
+		for _, pod := range pods {
+			logrus.Infof("[DEBUG]: ns: %v, name: %v", pod.GetNamespace(), pod.GetName())
```

Comment:
nit: could we remove this debug message?

Reply:
yes, we should... nice catch! updated!